components/channel_notifications_modal/channel_notifications_modal.jsx

```diff
@@ -640,10 +649,3 @@ export default class ChannelNotificationsModal extends React.Component {
     }
 }
 
```

Comment:
Please update spelling of `global` from ``// Get glabal user setting for notifications``